src/react/databrowser/objects/ObjectDelete.tsx

```diff
@@ -137,29 +202,67 @@ const ObjectDelete = ({
       isOpen={true}
       title="Confirmation"
     >
-      <div> {title(toggled, bucketInfo.isVersioning)} </div>
+      <div> {title(toggledFiles, bucketInfo.isVersioning)} </div>
       <Files>
         <Table>
           <T.Body>
-            {toggled.map((s) => (
+            {toggledFiles.map((s) => (
               <T.Row key={s.key}>
                 <T.Cell>
                   {' '}
                   {s.key}
                   <VersionId hidden={!s.versionId}> {s.versionId} </VersionId>
                   <Description>
                     {s.size && <PrettyBytes bytes={s.size} />}
+                    {(s.lockStatus === "LOCKED") && <div
+                      style={{
+                        color: theme.brand?.textTertiary,
+                      }}
+                    >
+                      <Icon style={{ display: 'inline', marginRight: '0.57rem' }} className="fa fa-lock"></Icon>
+                      {confirmed ? <span>Protected, will be deleted</span> : <span>Protected, will not be deleted</span>}
+                    </div>}
                   </Description>
                 </T.Cell>
+                <RemoveCell>
+                  {' '}
+                  <div onClick={() => removeFile(s.key)}>
+                    {' '}
+                    <i className="fas fa-times"></i>{' '}
+                  </div>{' '}
+                </RemoveCell>
               </T.Row>
             ))}
           </T.Body>
         </Table>
       </Files>
-      <div>
+      <div style={{ marginBottom: '1.25rem' }}>
         {' '}
         Total: <PrettyBytes bytes={totalSize} />{' '}
       </div>
+      {hasLockedFiles && <Fragment>
+      <div style={{ width: '31.25rem' }}>
+      <Banner
+        icon={<i className="fas fa-exclamation-circle" />}
+        variant="warning"
+      >
+        At least one object you want to delete is under <br />
+        Object-Lock retention <Icon style={{ display: 'inline', marginLeft: '0.063rem', marginRight: '0.063rem' }} className="fa fa-lock"></Icon> with governance mode.
```

Comment:
I would try to use `spacebox` from core-ui to have a cleaner code.
The purpose of spacebox is to define the margins by using styled-component.